Description: fix "Can't use string..." as an ARRAY ref at line 160
 Not sure what other problems this creates, but it seems to fix the
 problem at hand, which was making dizzy completely unusable.
Author: Damyan Ivanov <dmn@debian.org>
Bug-Debian: https://bugs.debian.org/730737
Forwarded: yes

--- a/lib/Dizzy/Perl2GLSL.pm
+++ b/lib/Dizzy/Perl2GLSL.pm
@@ -50,10 +50,8 @@ sub walk_optree {
 	} elsif ($optype eq "OP") {
 		if ($op->name eq "padsv") {
 			return "var" . $op->targ;
-		} elsif ($op->name eq "pushmark" or $op->name eq "null") {
-			return ();
 		} else {
-			return "# op description " . $op->desc;
+			return ();
 		}
 	} elsif ($optype eq "COP") {
 		return ();
